Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -33,32 +33,27 @@ public class BTShopperInsightsClient {
     public func getRecommendedPaymentMethods(request: BTShopperInsightsRequest) async throws -> BTShopperInsightsResult {
         apiClient.sendAnalyticsEvent(BTShopperInsightsAnalytics.recommendedPaymentsStarted)
         
-        if isVenmoAppInstalled() && isPayPalAppInstalled() {
-            let result = BTShopperInsightsResult(isPayPalRecommended: true, isVenmoRecommended: true)
-            return notifySuccess(with: result)
-        } else {
-            // TODO: - Fill in appropriate merchantID (or ppClientID) from config once API team decides what we need to send
-            let postParameters = BTEligiblePaymentsRequest(
-                email: request.email,
-                phone: request.phone,
-                merchantID: "MXSJ4F5BADVNS"
-            )
-            
-            do {
-                let (json, _) = try await apiClient.post("/v2/payments/find-eligible-methods", parameters: postParameters, httpType: .payPalAPI)
-                guard let eligibleMethodsJSON = json?["eligible_methods"].asDictionary(),
-                      eligibleMethodsJSON.count != 0 else {
-                    throw self.notifyFailure(with: BTShopperInsightsError.emptyBodyReturned)
-                }
-                let eligiblePaymentMethods = BTEligiblePaymentMethods(json: json)
-                let result = BTShopperInsightsResult(
-                    isPayPalRecommended: isPaymentRecommended(eligiblePaymentMethods.paypal),
-                    isVenmoRecommended: isPaymentRecommended(eligiblePaymentMethods.venmo)
-                )
-                return self.notifySuccess(with: result)
-            } catch {
-                throw self.notifyFailure(with: error)
+        // TODO: - Fill in appropriate merchantID (or ppClientID) from config once API team decides what we need to send
+        let postParameters = BTEligiblePaymentsRequest(
+            email: request.email,
+            phone: request.phone,
+            merchantID: "MXSJ4F5BADVNS"
+        )
+        
+        do {
+            let (json, _) = try await apiClient.post("/v2/payments/find-eligible-methods", parameters: postParameters, httpType: .payPalAPI)
+            guard let eligibleMethodsJSON = json?["eligible_methods"].asDictionary(),
+                  eligibleMethodsJSON.count != 0 else {
```

Comment:
I think we can use `!eligibleMethodsJSON.isEmpty` here instead if we want

Reply:
Hm - since `eligibleMethodsJSON` is an `NSDictionary`, we run into `Value of type 'NSDictionary' has no member 'isEmpty'`